src/components/Link/SmartLink.tsx

```diff
@@ -31,21 +31,26 @@ export function SmartLink({
   )
 }
 
-interface SmartLinkProps {
-  href: string
+export interface SmartLinkProps {
   children: ReactNode
+  href?: string
```

Comment:
@Schwartz10 I made `href` optional, as it's not required when using the `download` attribute